packages/contracts/package.json

```diff
@@ -8,9 +8,12 @@
     "hardhat": "^2.7.1"
   },
   "scripts": {
-    "test": "hardhat test",
-    "test:slither": "slither .",
-    "build": "hardhat compile",
+    "build": "forge build --root . -c contracts --out forge-artifacts",
+    "test:hh": "yarn hardhat test",
+    "test:forge": "forge test --root . -c contracts --out forge-artifacts",
```

Comment:
The one janky thing I had to do to make this work was setup separate artifacts for each testing framework, because forge writes the artifacts into a different dir layout than hh does. 

I think it's fine for now.

Reply:
So currently `yarn build && yarn test:hh` does not work, right?

Reply:
I think test:hh actually compiles if it doesn't find the artifacts.